packages/page-staking/src/Actions/Account/ListNominees.tsx

```diff
@@ -17,6 +19,29 @@ function ListNominees ({ nominating, stashId }: Props): React.ReactElement<Props
   const { t } = useTranslation();
   const { api } = useApi();
   const { nomsActive, nomsChilled, nomsInactive, nomsOver, nomsWaiting } = useInactives(stashId, nominating);
+  const sessionInfo = useCall<DeriveSessionProgress>(api.query.staking && api.derive.session?.progress);
+  const eraExposure = useCall<DeriveEraExposure>(api.derive.staking.eraExposure, [sessionInfo?.activeEra]);
```

Comment:
This is one of those problematic ones that may break on Edgeware, it doesn't have that functionality. (Not sure if the derive itself does the check, it should...)

Reply:
Oh thanks, checking that (and will add it in the derive if it's not there)